src/com/owncloud/android/ui/fragment/ExtendedListFragment.java

```diff
@@ -327,6 +357,23 @@ public void setSwipeEnabled(boolean enabled) {
     }
 
     /**
+     * Disables FAB.
```

Comment:
No worries, I like your eyes for the details! :+1: 
Done, see https://github.com/owncloud/android/commit/4327cf5433744611248630b48c49dbcee04048e3